src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,21 @@
-export const UserInfo = () => {};
+import React from 'react';
+import './UserInfo.scss';
```

Comment:
Consider importing styles using CSS modules for better scoping and maintainability. 💅